asyncsleepiq/fuzion/foundation.py

```diff
@@ -111,6 +115,14 @@ async def init_foot_warmers(self) -> None:
             if result == "1":
                 self.foot_warmers.append(SleepIQFuzionFootWarmer(self._api, self.bed_id, side, 0, 0))
 
+    async def init_heidi_modes(self) -> None:
+        """Initialize list of heidi modes available on foundation."""
+        for side in [Side.LEFT, Side.RIGHT]:
+            print(side)
+            result = await self._api.bamkey(self.bed_id, "GetHeidiPresence", args=[SIDES_FULL[side].lower()])
+            if result == "true":

```

Comment:
Does this return "true" or "1"?

Reply:
It returns "true" not "1". 

I found that during testing.